src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -310,6 +311,9 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
             for (int spaceNum : state.dataHeatBal->Zone(Loop).spaceIndexes) {
                 state.dataHeatBal->spaceAirRpt(spaceNum).setUpOutputVars(
                     state, DataStringGlobals::spacePrefix, state.dataHeatBal->space(spaceNum).Name);
+                std::string const &spaceName = state.dataHeatBal->space(spaceNum).Name;
+                auto &thisSpaceAirRpt = state.dataHeatBal->spaceAirRpt(spaceNum);
+                SetupEMSInternalVariable(state, "Space Wetbulb Globe Temperature", spaceName, "[C]", thisSpaceAirRpt.WetbulbGlobeTemp);
```

Comment:
This is not needed. Since the new WBGT is an output:variable is can be accessed with EMS:Sensor. The internal variable setup is to provide EMS access to values that are not time series data such as sizing results. See EMS Applicatoin Guide "Internal Variables" section.

Reply:
If I don't add this, it doesn't show up in the edd file. Is that expected?

Reply:
> If I don't add this, it doesn't show up in the edd file. Is that expected?

Correct. The EDD will list available actuators and internal variables only. Everything in the RDD is available to use as an EMS sensor.

Reply:
Thanks for the clarification. I will remove the SetupEMSInternalVariables.